python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_l2vlans.schema.yml

```
@@ -115,9 +115,10 @@ $defs:
               description: Will be enabled automatically if evpn_l2_multicast is enabled.
             source_address:
               type: str
-              description: |
-                IPv4_address
-                If not set, IP address of "Loopback0" will be used.
+              description: |-
+                Since there is no VRF tied to an L2VLAN, the values  `vrf_router_id`, `diagnostic_loopback` and `main_router_id` will all configure the Loopback0 IP address.
```

```suggestion
                Since there is no VRF tied to an L2VLAN, the values `vrf_router_id`, `diagnostic_loopback` and `main_router_id` will all configure the Loopback0 IP address.
```

---

Need to explain the values in here and explain why they all point to Loopback0:


```suggestion
                The value of `source_address` will be interpreted according to these rules:
                - `vrf_router_id`, `diagnostic_loopback` and `main_router_id` will configure the Loopback0 IP since there is no VRF tied to an L2VLAN.
                - An IPv4 address will be used directly as the source address.
```

---

updated